cvat/apps/engine/utils.py

```diff
@@ -160,20 +193,33 @@ def configure_dependent_job(
     key: str,
     request: HttpRequest,
     result_ttl: float,
-    failure_ttl: float
+    failure_ttl: float,
+    should_be_dependent: bool = settings.ONE_RUNNING_JOB_IN_QUEUE_PER_USER,
 ) -> Job:
-    rq_job_id_download_file = rq_id + f'?action=download_{filename}'
+    rq_job_id_download_file = rq_id + f'?action=download_{key.replace("/", ".")}'
```

Comment:
Could you comment on this `replace` change? Why it needed? It can lead to task name collisions with some filenames with dots (in all invocations key is taken from filename).

Reply:
How it can lead to task name collisions if rq_id is unique per import operation?
Replace is required because a key can contain a forward slash that is not an acceptable symbol for rq id generally because it leads to the inability to open page with rq job details (django-rq/queues/{id}/{rq_id}/)

Reply:
So what happens if we have a single task creation request with 2 files on a cloud storage:
```
a/b.ext
a.b.ext
```

Can the jobs collide?